tests/unit/test_protocol_errors.py

```diff
@@ -22,6 +22,7 @@ class TestFrontendErrors(unittest.TestCase):
     """
     @classmethod
     def setUpClass(cls):
+        reload(frontend)
```

Comment:
Nothing has changed, but previously all the tests used the same configuration. Since the flask app is a module-level variable of frontend, it's instantiated when the module is loaded, and then you can configure() it. But it turns out that running `frontend.configure(...)` twice doesn't work (since the app has started by that point). So we reload the module to recreate the flask app.